testdata/bad_customtemplates/function.tmpl

```diff
@@ -41,7 +41,7 @@ func {{.TestName}}(t *testing.T) {
 	}{
 		// TODO: Add test cases.
 	}
-	for {{if not .IsNaked}} _, tt := {{end}} range testCases {
+	for {{if (or .Subtests (not .IsNaked))}} _, tt := {{end}} range tests {
```

Comment:
I believe it was intentional for `tests` to be renamed to `testCases` here, because these are custom templates. Same below. Could you please revert back?

Reply:
Indeed, tests are failing: https://travis-ci.org/cweill/gotests/jobs/446258506.

Reply:
Good call. fixed